### ISSUE TITLE
Investigate ways to identify maintainers

### ISSUE THREAD

**Issue by jknust** [kalepeterson/InetSysDev-GH-Community-Health#8]
Looks for distinguishing attributes that would signify that a user is a maintainer for a specific repo.

**Comment by kalepeterson** [comment]
After looking into this a couple times this semester, we are pretty sure there's not a straightforward way of doing this with GHTorrent data.  Identifying project members can work, but the users in the project members table can be non-maintainers as well.  Analyzing issue data could produce _something_ close to identifying maintainers, but we are not convinced this would be an exact science either.

In summary, we have determined that identifying maintainers is very difficult and fuzzy, if not outright impossible.
